nulib/dc-nextjs
# Integrate Cypress End to End Test starter

## What does this do?
- Sets up the framework for running E2E tests against the [DC Test Environment](https://github.com/nulib/dc-test-environment) from a dev's local laptop DC v2 dev environment (not sure this would work in our AWS space yet)
- Implements E2E tests against the Search page
- Implements starter E2E tests against the Home page

## How to Test
- Run the E2E tests locally (see the README, as this will prove whether the instructions are complete or clear enough)
